### PR TITLE
🧹🧹 Add issue templates 

### PR DESCRIPTION
**Pull request by kolibril13**
Follow up to https://github.com/executablebooks/jupyterlab-myst/pull/130.
The templates are quite minimalistic, which I think is good to lower the contribution barrier.
@agoose77 , @rowanc1 

EDIT: A preview can be found in a dummy repo here:
https://github.com/kolibril13/okapi2/issues/new/choose

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Questions or general discussion with the community
+    url: https://github.com/executablebooks/meta/discussions
+    about: Use our Community Forum for general conversations that aren't meant for actionable Issues.
```
Comment (rowanc1) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    about: Use our Community Forum for general conversations that aren't meant for actionable issues.
```

**Review by rowanc1 — APPROVED** [review]
Looks good and super simple. :)

I will let @agoose77 comment tomorrow and we can get this going. Can always tweak it in the future! 🚀

Thanks @kolibril13!

**Comment by rowanc1** [comment]
Apparently the linter looks at all markdown files! Can you fix the linting @kolibril13?

**Comment by kolibril13** [comment]
done! 👍🏼

**Comment by rowanc1** [comment]
Thanks @kolibril13, appreciate the help over in this repo as well!! :)

**Pull request merged**